InformationScripting/src/parsing/QueryBuilder.cpp

```diff
@@ -81,12 +81,13 @@ QPair<QStringList, QList<QChar>> QueryBuilder::split(const QString& text, const
 	QPair<QStringList, QList<QChar>> result;
 	QString currentString;
 	QVector<int> openScopes(3, 0);
-	for (int i = 1; i < text.size() - 1; ++i)
+	for (int i = 2; i < text.size() - 2; ++i)
```

Comment:
There are a lot of magical numbers here. 2 should be inferred somehow, from the OPEN/CLOSE_SCOPE_SYMBOL, not a hardcoded constant.